pf-freebsd-setup/azuredeploy.json

```diff
@@ -0,0 +1,342 @@
+      }
+    },
+    "backendVM2PrivateNicIP": {
+      "type": "string",
+      "defaultValue": "10.0.2.7",
+      "metadata": {
+        "description": "Backend VM2 private IP address which resides in private subnet."
+      }
+    },
+    "frontendVmSize": {
+      "type": "string",
+      "defaultValue": "Standard_A4",
+      "metadata": {
+        "description": "Frontend VM size, please choose proper size which allow 2 NICs."
+      }
+    },
+    "frontendVmStorageAccountType":{
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "Type of frontend VM storage account."
+      }
+    },
+    "backendVmSize": {
+      "type": "string",
+      "defaultValue": "Standard_A1",
+      "metadata": {
+        "description": "Backend VM size."
+      }
+    },
+    "backendVmStorageAccountType":{
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "Type of backend VM storage account."
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "defaultValue": "azureuser",
+      "metadata": {
+        "description": "User name for the Virtual Machine"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for frontend and backend VMs."
+      }
+    },
+    "vnetAddressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/19",
+      "metadata": {
+        "description": "Address prefix for VNET."
+      }
+    },
+    "publicSubnetAddressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/23",
+      "metadata": {
+        "description": "Address prefix for public subnet."
+      }
+    },
+    "privateSubnetAddressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.2.0/23",
+      "metadata": {
+        "description": "Address prefix for private subnet."
+      }
+    },
+    "networkPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Network prefix."
+      }
+    },
+    "domainNamePrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "environment": {
+      "type": "string",
+      "defaultValue": "AzureCloud",
+      "allowedValues": [
+        "AzureCloud",
+        "AzureChinaCloud"
+      ],
+      "metadata": {
+        "description": "Different environments in Azure. Choose AzureCloud for Global Azure, choose AzureChinaCloud for Azure China Cloud."
+      }
+    },
+    "clientID": {
+      "type": "string",
+      "minLength": 36,
+      "maxLength": 36,
+      "metadata": {
+        "description": "ID of the client. See https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/blob/master/docs/guidance.md"
```

Comment:
Yes, this document comes from our team member, it contains script that we can get service principal easily.